Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -497,6 +497,53 @@ function Update-IdentitySourceCertificates {
     }
 }
 
+<#
+    .Synopsis
+     Update the password used in the credential for authenticating to an Active Directory over LDAPS
+
+    .Parameter Credential 
+     Credential to login to the LDAP server (NOT cloudadmin) in the form of a username/password credential. Usernames often look like prodAdmins@domainname.com or if the AD is a Microsoft Active Directory server, usernames may need to be prefixed with the NetBIOS domain name, such as prod\AD_Admin
+
+    .Parameter DomainName
+     Domain name of the external active directory, e.g. myactivedirectory.local
+#>
+function Update-IdentitySourceCredentialPassword {
+    [CmdletBinding(PositionalBinding = $false)]
+    [AVSAttribute(10, UpdatesSDDC = $false)]
+    Param
+    (
+        [Parameter(
+                Mandatory = $true,
+                HelpMessage = 'Name of the Identity source')]
+        [ValidateNotNull()]
+        [string]
+        $DomainName,
```

Comment:
I think we should support this being optional but we can check with Joe and Eugene.
If there is nothing provided we attempt to rotate the credentials of the only AD they have configured. If they have more than one, then we should return an error telling them they have to specify

Reply:
@et1975 should we support optional?